module/VuFind/src/VuFind/Controller/LibraryCardsController.php

```diff
@@ -240,6 +242,35 @@ public function selectCardAction()
         return $this->redirect()->toRoute('myresearch-home');
     }
 
+    /**
+     * Redirects to Shibboleth authentication to connect a new library card
+     *
+     * @return \Laminas\Http\Response
+     */
+    public function connectNewShibbolethCardAction()
+    {
+        $url = $this->getServerUrl('librarycards-connectshibbolethcard');
+        $redirectUrl = $this->getAuthManager()->getSessionInitiator($url);
```

Comment:
Check that getSessionInitiator doesn't return false

Reply:
While checking for null with the `==` operator works, I'd rather just check `if (!$redirectUrl)` since getSessionInitiator should only return false or a string and the check would cater for both false and an empty string.